packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -5609,9 +5609,9 @@ namespace Tpetra {
         os << *prefix << "Target is StaticProfile; do CRS padding" << endl;
         std::cerr << os.str ();
       }
-      auto padding = computeCrsPaddingNew (srcRowGraph, numSameIDs,
-                                           permuteToLIDs, permuteFromLIDs);
-      this->applyCrsPadding (padding);
+      auto padding =
```

Comment:
This is fine for now, but in the future, we should have a special case for when the target has a const graph.  If so, then we don't need to compute padding at all, since the target's structure can't change.